cdap-client/src/main/java/io/cdap/cdap/client/util/RESTClient.java

```diff
@@ -167,9 +168,11 @@ private boolean isSuccessful(int responseCode) {
   }
 
   private Map<String, String> getAuthHeaders(AccessToken accessToken) {
-    Map<String, String> headers = ImmutableMap.of();
+    Map<String, String> headers = clientConfig.getAdditionalHeaders();
     if (accessToken != null) {
-      headers = ImmutableMap.of(HttpHeaders.AUTHORIZATION, accessToken.getTokenType() + " " + accessToken.getValue());
+      headers = new HashMap<>(headers);
```

Comment:
nit: you don't need to make an extra map, you can use a ImmutableMap builder

Reply:
Fixed, thanks!